README.md

```diff
@@ -1 +1,23 @@
-# company-search-consumer
\ No newline at end of file
+# company-search-consumer
+A service that consumes messages from the stream-company-profile topic and then sends a RESTful POST request to the search.api.ch.gov.uk service. The data sent through the POST request will be used in search.api.ch.gov.uk to maintain and update elastic search indexes relating to company search.
```

Comment:
The last sentence of this summary is stating the behaviour of a different service. If the other service changes, this documentation will be incorrect. Can this summary be a high level explanation of the purpose of this consumer only please?